### PR TITLE
[Mobile app] Fix - failure on startup

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)

- [x] Adds exception handling to secure storage.
- [x] Temporarily disables notification listening.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [MOB-126](https://airqoteam.atlassian.net/browse/MOB-126)

#### Screenshots (optional)

### CODE REVIEW

File: mobile/lib/services/secure_storage.dart [hunk 1]
```diff
@@ -8,30 +8,63 @@ class SecureStorage {
...
+    }
   }
 
-  Future<String?> getValue(String key) {
-    return _secureStorage.read(key: key);
+  Future<String?> getValue(String key) async {
+    try {
+      return _secureStorage.read(
+        key: key,
+        aOptions: _getAndroidOptions(),
```
Comment (NoahNsimbe) [inline mobile/lib/services/secure_storage.dart:34]: Setting android encryption options.
Ref: https://pub.dev/packages/flutter_secure_storage

File: mobile/lib/services/hive_service.dart [hunk 2]
```diff
@@ -36,27 +36,33 @@ class HiveService {
...
+      if (encodedKey == null) {
+        final secureKey = Hive.generateSecureKey();
+        await secureStorage.setValue(
+          key: HiveBox.encryptionKey,
+          value: base64UrlEncode(secureKey),
+        );
+      }
+      encodedKey = await secureStorage.getValue(HiveBox.encryptionKey);
+
+      return base64Url.decode(encodedKey!);
```
Comment (NoahNsimbe) [inline mobile/lib/services/hive_service.dart:62]: Adding error handling to encryption key

File: mobile/lib/main_common.dart [hunk 3]
```diff
@@ -59,3 +61,15 @@ class AppHttpOverrides extends HttpOverrides {
...
   }
 }
+
+Future<void> initializeMainMethod() async {
+  await Future.wait([
+    SystemProperties.setDefault(),
+    dotenv.load(fileName: Config.environmentFile),
+    HiveService.initialize(),
+    // NotificationService.listenToNotifications(),
+    // initializeBackgroundServices()
```
Comment (NoahNsimbe) [inline mobile/lib/main_common.dart:71]: Temporarily disabling background services

**Review by Baalmart — APPROVED** [review]
This looks good to me. Thanks @NoahNsimbe

**Pull request merged**